ethereum/contracts/ETHApp.sol

```diff
@@ -72,6 +78,10 @@ contract ETHApp is RewardController, AccessControl {
         _setupRole(INBOUND_CHANNEL_ROLE, _incentivized.inbound);
     }
 
+    receive() external payable {
```

Comment:
This will allow the vault to transfer funds when calling `unlock`.

Reply:
Hmm, I think its simpler to make the vault give the funds directly to the user, rather than passing the funds upwards to the app and to the user.